demos/end-to-end-security/create-spark-report.yaml

```
@@ -0,0 +1,159 @@
+# We can't simply create the SparkApplication object here as we have to wait for Kafka to be ready because
+# * We currently don't restart failed Spark applications (see https://github.com/stackabletech/spark-k8s-operator/issues/157)
+# * We currently auto-create topics and we need all the brokers to be available so that the topic is distributed among all the brokers
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: create-spark-report
+spec:
+  template:
+    spec:
+      serviceAccountName: demo-serviceaccount
+      initContainers:
+        - name: wait-for-trino-tables
+          image: docker.stackable.tech/stackable/tools:1.0.0-stackable24.3.0
+          command: ["bash", "-c", "echo 'Waiting for Job create-tables-in-trino to complete' && kubectl wait --timeout=30m --for=condition=complete job/create-tables-in-trino"]
+      containers:
+        - name: create-spark-report
+          image: docker.stackable.tech/stackable/tools:1.0.0-stackable24.3.0
+          command: ["bash", "-c", "echo 'Submitting Spark report' && kubectl apply -f /tmp/manifest/spark-report.yaml"]
```

nit: I personally find it easier to read when the command is broken into a yaml list (removing baces, quotes, multi-line):

I'll leave it up to you if you want to commit this.

```suggestion
      initContainers:
        - name: wait-for-trino-tables
          image: docker.stackable.tech/stackable/tools:1.0.0-stackable24.3.0
          command:
          - bash
          - -c
          - |
            echo 'Waiting for Job create-tables-in-trino to complete'
            kubectl wait --timeout=30m --for=condition=complete job/create-tables-in-trino
      containers:
        - name: create-spark-report
          image: docker.stackable.tech/stackable/tools:1.0.0-stackable24.3.0
          command:
          - bash
          - -c
          - |
            echo 'Submitting Spark report'
            kubectl apply -f /tmp/manifest/spark-report.yaml
```

---

Makes sense, please have a look at 07a89fb048561f1dad7faf46eb790c1492430562